app/javascript/react/redux/greetings/greetingSlice.js

```diff
@@ -0,0 +1,33 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+export const fetchGreetings = createAsyncThunk('message/fetch', async () => {
+  const response = await fetch('/api/v1/messages');
+  const data = await response.json();
+  //const message = data.message;
+  console.log(data)
+  //return message;
```

Comment:
- [ ] In order for your thunk to work, you need to return a value which is going to be your `action.payload` in the reducer, I see that you had it but left it commented, please remove the console statement and uncomment line 6 and 8 for the thunk to work! 🤓 🔧